<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTest2</class>
 <widget class="QWidget" name="FormTest2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="6" rowspan="4">
    <widget class="QSlider" name="verticalSlider">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <widget class="QDial" name="dial"/>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Foo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Baz</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="4" column="5">
    <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
